control-plane/rest/service/src/v0/snapshots.rs

```diff
@@ -78,7 +78,8 @@ impl apis::actix_server::Snapshots for RestApi {
             ),
             state: models::VolumeSnapshotState::new_all(
                 snap.state().uuid(),
-                snap.state().size(),
+                // todo: should be optional?
+                snap.state().size().unwrap_or_default(),
```

Comment:
Optional Should be better I guess, rather than showing 0 ?

Reply:
Yeah I think so to